src/js/modules/SectionHeader.js

```diff
@@ -1,8 +1,19 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import ArrowRight from 'js/svg/ArrowRight';
 
-const SectionHeader = ({ title }) => (
-  <h2 className="coa_SectionHeader">{title}</h2>
+
+const SectionHeader = ({ title, arrow }) => (
```

Comment:
also picky -- for boolean props like arrow, can we make the name obvious that the prop is a bool. Like, hasArrow?